packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -87,6 +105,12 @@ class ToOOXMLVisitor {
                 propertyTag += LINK_PROPERTY_RULE();
             }
         }
+        if (this.isTraversingFalseCondition) {
+            propertyTag += '<w:vanish/>';
+        }
+        if (this.isTraversingElseCondition) {
+            propertyTag += '<w:rFonts w:ascii="Baskerville Old Face" w:hAnsi="Baskerville Old Face"/>';
```

Comment:
This should be in the rules file. You have put most properties and nodes in `rules.js` so why have you made an exception here?